### PR TITLE
feat: add unstitched second factor mutations

### PR DESCRIPTION
**Pull request by mzikherman**
The start...many more mutations to come!

### CODE REVIEW

File: src/lib/stitching/gravity/schema.ts [hunk 1]
```diff
@@ -98,5 +108,14 @@ export const executableGravitySchema = () => {
         return name
       }
     }),
+    new FilterRootFields((operation, name) => {
```
Comment (mzikherman) [inline src/lib/stitching/gravity/schema.ts:111]: This is new, we want to start excluding certain mutations from Gravity's schema.

File: src/schema/v2/me/secondFactors/mutations/createSmsSecondFactor.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      type: GraphQLString,
+    },
+    phoneNumber: {
+      type: GraphQLString,
+    },
+  },
+})
+
+const ErrorType = new GraphQLObjectType<any, ResolverContext>({
+  name: "Error",
```
Comment (mzikherman) [inline src/schema/v2/me/secondFactors/mutations/createSmsSecondFactor.ts:26]: This is interesting...basically nothing complains that this is a duplicate type, schema generation and resolution looks correct - even though this type is defined in Gravity as well.

This is necessary b/c clients do `... on Errors` in the second factor mutation.

This is a bit weird, but I'll keep this pattern going in an effort to unstitch with a no-op.

File: src/schema/v2/me/secondFactors/mutations/createSmsSecondFactor.ts [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+      const data = await createSecondFactorLoader({
+        kind: "sms",
+        password,
+        attributes: snakeCaseAttributes,
+      })
+
+      return data[0]
+    } catch (error) {
+      const { body } = error
```
Comment (mzikherman) [inline src/schema/v2/me/secondFactors/mutations/createSmsSecondFactor.ts:101]: To abide by the format of these mutation errors (slightly different than our preferred pattern), I'll just drop in a simple helper to format as needed.

File: src/schema/v2/me/secondFactors/mutations/createSmsSecondFactor.ts (hunk 2 above)
Comment (damassi) [reply src/schema/v2/me/secondFactors/mutations/createSmsSecondFactor.ts:26]: Super weird, but sweet 😎

**Review by damassi — DISMISSED** [review]

File: _schemaV2.graphql [hunk 4]
```diff
@@ -1162,27 +1162,33 @@ type AnalyticsVisitorsByReferral {
   value: Int!
 }
 
-# App Authenticator Two-Factor Authentication factor
```
Comment (mzikherman) [inline _schemaV2.graphql:1165]: This diff should be safe for clients - `yarn relay` results in zero changes with this diff.

**Comment by mzikherman** [comment]
Merging this in to get the schema pushed to clients 🤞

File: src/lib/stitching/gravity/schema.ts [hunk 5]
```diff
@@ -54,8 +55,43 @@ export const executableGravitySchema = () => {
...
+  const excludedMutations: string[] = []
+  if (config.USE_UNSTITCHED_SECOND_FACTORS_SCHEMA) {
+    excludedMutations.push("createSmsSecondFactor")
+    excludedMutations.push("updateSmsSecondFactor")
+    excludedMutations.push("createAppSecondFactor")
+    excludedMutations.push("updateAppSecondFactor")
+    excludedMutations.push("createBackupSecondFactors")
+    excludedMutations.push("disableSecondFactor")
+    excludedMutations.push("deliverSecondFactor")
+    excludedMutations.push("enableSecondFactor")
```
Comment (damassi) [inline src/lib/stitching/gravity/schema.ts:94]: 💥

**Pull request merged**